dev/release/setup-ubuntu.sh

```diff
@@ -0,0 +1,33 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# A script to install dependencies required for release
+# verification on Ubuntu 20.04
+
+apt-get update
+apt-get -y install \
+  build-essential \
+  openjdk-8-jdk \
+  pkg-config \
+  maven \
+  ruby-dev \
+  cmake \
+  python3-pip \
+  ninja-build \
+  llvm-dev \
+  clang \
+  libssl-dev
```

Comment:
Could you sort this list in alphabetical order?

Reply:
ok